Lesson 2/EndOfLesson2.java

```diff
@@ -0,0 +1,16 @@
+import java.util.*;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+public class EndOfLesson2 {
+    public static void main(String[] args) {
+        EndOfLesson2.showNameAndTime("Eugene", "Bujanov");
```

Comment:
Name and Surname are hardcoded rights now, I would like as a User enter name and surname by myself

Reply:
added EndOfLesson2ver3.java where it is possible to pass Name and Surname from the command line

Reply:
I would suggest you need to google up method or methods how to read input from console...

Reply:
In "Files changed" tab you can find EndOfLesson2ver2.java where program asks to input name and surname using Scanner and also EndOfLesson2ver3.java where we can by running program in console pass argument with our Name and Surname to console. Example to run in console: java EndOfLesson2ver3 "Eugene Buyanov".